subprojects/gradle/module-dependencies-graph/src/gradleTest/kotlin/com/avito/module/dependencies/InMemoryProjectGenerator.kt

```diff
@@ -0,0 +1,35 @@
+package com.avito.module.dependencies
+
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+
```

Comment:
I've used ProjectBuilder to test logic.
It requires to make something kinda TestProjectGenerator. I'll file an issue.

But even in this case, it's worth it. We don't have to wait for slow GradleTestKit and parse the output.
It helps to check much faster.